lib/ngx/re.lua

```diff
@@ -87,11 +87,18 @@ local function re_split_helper(subj, compiled, compile_once, flags, ctx)
         return nil, nil, nil
     end
 
+    if from == to then
+      -- empty match, skip to next char
+      ctx.pos = to + 1
+
+    else
+      ctx.pos = to
```

Comment:
Please use 4-space indentations consistently.

Reply:
Sorry, pushed this fix from an usual setup while on holidays, will update.